docs/OpenTypeFeatureFileSpecification.md

```diff
@@ -0,0 +1,3769 @@
+OpenType™ Feature File Specification
+---
+
+Copyright 2015-2019 Adobe. All Rights Reserved. This software is licensed as
+OpenSource, under the Apache License, Version 2.0. This license is available at:
+http://opensource.org/licenses/Apache-2.0.
+
+Document version 1.24  
+Last updated 20 March 2019
+
+**Caution: Portions of the syntax unimplemented by Adobe are subject to change.** 
+
+## Contents
+
+1.  [Introduction](#1)
+2.  [Syntax](#2)
+
+    1.  [Comments](#2.a)
+    2.  [White space](#2.b)
+    3.  [Keywords](#2.c)
+    4.  [Special characters](#2.d)
+    5.  [Numbers and other metrics](#2.e)
```

Comment:
Instead of the manual anchors, you can use `[Introduction](#introduction)`, `[Numbers and other metrics](#numbers-and-other-metrics)`, etc.

Reply:
I have though about and considered that, however there are so many cross-references that those would introduce a lot of noise in the raw markdown file.